### DISCUSSION TITLE
Group by name and display last record

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#3874] [Help] [answered]
Hi Team, 
I am having trouble figuring this one out. I am hoping someone can point me in the right direction. 
I am using the internal BB DB I would like to group my records by the person's name and display only their last record is this possible?

**Comment by mjashanks** [comment, marked as answer]
Hi!

To do this, use nested repeater blocks:
1. Add a `Repeater Block` for your `Users` table
2. Underneath, add a `Repeater block` for the relationship: `Repeater Block.Users.YourRelationShip`
3. On the 2nd repeater block, set the limit to 1

If you want to use a table for the second repeater block, use 
```
>  Data Provider : Repeater Block.Users.YourRelationShip
   > Table
```

**Reply by KernOil** [reply]
is there a way for me to sort a repeater by a calculated number? in the data tab, I have a formula column when I try and sort by that number in the repeater it turns blank. do you know how I can get around this?
